### ISSUE TITLE
all vs MW 3-grams, part 4

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/CORRECTIONS#288]
#284 continued.
Total 84 entries to examine.

**Comment by drdhaval2785** [comment]
Not found
```

```

**Comment by drdhaval2785** [comment]
301-310
```
pui:vAqohali,12124:vAqohali:n:qoh
vcp:vAmalovanA,41100:vAmalocanA:t:lov
sch:vAraRabUsA,23525:vAraRabusA:p:UsA. All other dictionaries have vAraRabusA. Also next entry has vAraRavusA, with the same meaning. no 'U' there.
sch:vAraRavusA,23526:vAraRavusA:n:avu,vus
bur:vAvrajmi,15118:vAvrajmi:n:jmi
bhs:vASIcandanakalpa,13562:vASIcandanakalpa:n:SIc
pui:vAsiKyas,12653:vAsiKyas:n:siK
bhs:vAsIcandanakalpa,13578:vAsIcandanakalpa:n:sIc
acc:viwuWalanATa,46617:viwWalanATa:t:iwu,wuW
bur:viDUnomi,15455:viDUnomi:n:Uno
```

**Comment by drdhaval2785** [comment]
311-320
```
shs:viDepada,36845:viDeyapada:t:Dep
acc:vivAhesaMdehadozOzaDa,41138:vivAhesaMdehadozOzaDa:n:ozO,hes
pui:viSvaSfjas,13244:viSvasfjas:p:Sfj is alphabetically misordered. Between two sfj.
bur:vizkaBnomi,15882:vizkaBnomi:n:Bno
bur:vistaBnomi,15922:vistaBnomi:n:Bno
inm:vudvudA,12391:vudvudA:n:dvu
sch:vfndAwavI,24384:vfndAwavI:n:dAw
pui:vERoya,13651:vERoya:n:Roy,ERo
pui:vEzWiya,13788:vEzWiya:n:Wiy
ap90:vogin,26910:vegin:t:vog
```

**Comment by drdhaval2785** [comment]
321-330
```
vcp:voSi,42584:voSi:n:voS
bhs:vosArayati,14612:vosArayati:n:vos
bur:vyADUnomi,16380:vyADUnomi:n:Uno
mci:Santanos,1722:Santanos:n:nos
bhs:SAWiya,14912:SAWiya:n:Wiy
acc:SAWyopanizad,24805:SAWyopanizad:n:Wyo
acc:SiGraboDa,47274:SIGraboDa:t:SiG
pui:SiGrya,14400:SiGrya:n:SiG
pe:SiyAli,6661:SiyAli:n:Siy
pui:SudDOdana,14640:SudDOdana:n:DOd
```

**Comment by drdhaval2785** [comment]
331-340
```
bhs:SuBesTitA,15137:SuBesTitA:n:Bes
sch:SfNgibera,25243:SfNgibera:n:gib
pui:SEByAs,14749:SEByAs:n:SEB
bhs:SEyyA,15184:SEyyA:n:SEy
pui:SElUja,14755:SElUja:n:lUj
pui:SElUsa,14756:SElUsa:n:lUs
ben:ScyuT,14952:Scyut:t:yuT
pw:SyOnaHSyepa,114797:SyOnaHSyepa:n:SyO
acc:SrIBImakAYcIvillIya,47423:SrIBImakAYcIvillIya:n:IBI
pui:SvaSfpa,15012:SvaSfpa:n:Sfp
```

**Comment by drdhaval2785** [comment]
341-350
```
sch:zaQAnya,25474:zaQAnya:n:zaQ
pwg:zIqIy,122427:zoqIy:t:zIq
mw72:saNgfByA,48566:saNgfByA:n:fBy
pw:saMGAdiSeza,117728:saMGAdiSeza:n:GAd
inm:saptEDas,9510:saptEDas:n:ptE
bhs:samantakusumArciHpralambacUqa,15875:samantakusumArciHpralambacUqa:n:ciH
bhs:samanteryApaTa,15960:samanteryApaTa:n:ery
bhs:saMpravutta,16282:saMpravutta:n:avu
wil:samPAsna,40935:samPAla:t:PAs
inm:sarvanAgaripuDvaja,9639:sarvanAgaripuDvaja:n:puD
```

**Comment by drdhaval2785** [comment]
351-360
```
mw72:sasfvas,51601:sasfvas:n:sfv
pui:sahEzikas,15719:sahEzikas:n:hEz
sch:sArastanIkEtavakOtuka,26463:sArastanIkEtavakOtuka:n:IkE
bur:sisIzAmi,18583:sisIzAmi:n:sIz
bur:sIzape,18615:sIzape:n:sIz
inm:suHSala,10213:suHSala:n:suH
acc:sudarSanasaMhitAyAMpaYcamuKihanumatkavaca,47876:sudarSanasaMhitAyAMpaYcamuKihanumatkavaca:n:Kih
yat:surayuTapAri,42124:surayUTapAri:t:yuT
pui:suvarRakewakas,16536:suvarRakewakas:n:kew
pgn:sUrInASIrampUrRRekakzetra,436:sUrInASIrampUrRRekakzetra:n:RRe
```

**Comment by drdhaval2785** [comment]
361-374
```
bhs:seKayati,17281:seKayati:n:seK
sch:sejavAlI,26944:sejavAlI:n:sej
pui:sEzirIwi,16760:sEzirIwi:n:sEz
pui:sOwi,16821:sOwi:n:sOw
pw:sOhva,130227:sOhva:n:Ohv
pwg:statipuWika,113679:stutipAWaka:p:puW
bur:steGayAmi,19169:steGayAmi:n:teG
bhs:sPUwa,17490:sPUwa:n:PUw
sch:svargesad,27313:svargesad:n:ges
shs:hawwavOraka,46487:hawwacOraka:t:vOr
wil:haraSesvarA,43961:haraSeKarA:t:Ses
yat:hfdayAbiD,44223:hfdayAviD:t:biD would be alphabetically misordered.
pw:hoSanikarRAwakavAYCeSvara,135621:hoSanikarRAwakavAYCeSvara:n:hoS
inm:hrdikasuta,4552:hfdikasuta:t:hrd
```

**Comment by drdhaval2785** [comment]
@funderburkjim 
AllvsMW_3grams is completed.
Now you can closed #284, #285, #286 and #288 with the file https://github.com/sanskrit-lexicon/CORRECTIONS/blob/master/ngram/output/corrections/allvsMW_3_corrected.txt.
All corrected entries are there in this file.

**Comment by gasyoun** [comment]
374 lines, hard work. Free Monier! Free Sanskrit!
